tests/foreman/destructive/test_rename.py

```diff
@@ -112,9 +113,18 @@ def test_positive_rename_satellite(module_org, module_product, module_target_sat
         'repository published path not updated correctly'
     )
 
-    # check for any other occurences of old hostname
-    result = module_target_sat.execute(f'grep " {old_hostname}" --exclude-dir="promtail" /etc/* -r')
-    assert result.status != 0, 'there are remaining instances of the old hostname'
+    # check config files (except certs/keys) for occurences of old hostname
+    result = module_target_sat.execute(
+        f'grep "{old_hostname}" /etc -r --exclude=template --exclude-dir={{promtail,pki}} --exclude=*.{{pem,cert,bak}}'
```

Comment:
Can you add a comment why those files and directories are excluded?
(I for example don't know why "template" is in here)

non-blocking: I find it easier to read w/o the bash-brackets
```suggestion
        f'grep "{old_hostname}" /etc -r --exclude=template --exclude-dir=promtail --exclude-dir=pki --exclude="*.pem" --exclude="*.cert" --exclude="*.bak"'
```
and you gotta quote those, as otherwise bash *migh* expand

Reply:
also, looking at an system here, there are also `.crt` files (not `.cert`) that have the old name (via the issuer)

and also:
```
grep: /etc/pki/ca-trust/extracted/edk2/cacerts.bin: binary file matches
grep: /etc/pki/ca-trust/extracted/java/cacerts: binary file matches
```
ignore "extracted"?

Reply:
might expand ? no it doesn't as it is part of the string

So,
`--exclude=*.pem` doesn't expand
while
`--exclude *.pem` does expand

Reply:
eww, right. eww ;)

Reply:
oh right :see_no_evil:

Reply:
Commented about `/etc/template`